docs/content/SelectPanel.mdx

```diff
@@ -66,4 +66,60 @@ function DemoComponent() {
 render(<DemoComponent />)
 ```
 
+## Interactions
```

Comment:
Could we call this section "Accessibility" to match the PageLayout docs: https://primer.style/react/PageLayout#accessibility
```suggestion
## Accessibility
```